numpydoc/docscrape.py

```diff
@@ -11,6 +11,10 @@
 import copy
 import sys
 
+if 'sphinx' in sys.modules:
+    from sphinx.ext.autodoc import ALL
```

Comment:
This logic needs to be used in ClassDoc, as globally the import order shouldn't matter

Reply:
Makes sense!